packages/twenty-server/test/integration/utils/create-app.ts

```diff
@@ -25,7 +28,21 @@ export const createApp = async (
 ): Promise<NestExpressApplication> => {
   let moduleBuilder: TestingModuleBuilder = Test.createTestingModule({
     imports: [AppModule],
-  });
+  })
```

Comment:
as discussed, let's initialize the app with the same parameters as the real app (i.e. cors true and rawbody true)